catppuccinpalette.dtx.tera

```diff
@@ -0,0 +1,362 @@
+---
+whiskers:
+    version: "2.5.0"
+    app: "LaTeX"
```

Comment:
You can remove this since it isn't used or required.

Reply:
You mean the whole block, or just the `whiskers.app`?

Because I think the `whiskers.version` is used to check if this version of the fileformat works with the current whiskers version.

Reply:
`whiskers.app`, and correct. GitHub has awful formatting here for this one selected line 😓